src/rust/engine/src/context.rs

```diff
@@ -143,16 +154,13 @@ impl Core {
           store.clone(),
         )),
         process_execution_remote_parallelism,
-      ),
-      None => BoundedCommandRunner::new(
-        Box::new(process_execution::local::CommandRunner::new(
-          store.clone(),
-          work_dir.clone(),
-          process_execution_cleanup_local_dirs,
-        )),
-        process_execution_local_parallelism,
-      ),
-    };
+      )
+    }
+
+    let command_runner = process_execution::speculate::SpeculatingCommandRunner::new(
```

Comment:
since you're bringing `SpeculatingCommandRunner` into scope you don't need these qualifications right?

Reply:
I'll fix the import